.github/workflows/labeler.yml

```diff
@@ -0,0 +1,13 @@
+name: "Pull Request Labeler"
+on:
+    schedule:
+        # * is a special character in YAML so you have to quote this string
+        - cron:  '*/5 * * * *'
```

Comment:
I'm afraid of this, as it will be launched all the time and we won't find in Actions list the other builds

Reply:
We can reduce this to one an hour.
I’ve set to 5 min to test.